src/test/java/organice/logic/LogicManagerTest.java

```diff
@@ -79,8 +79,9 @@ public void execute_storageThrowsIoException_throwsCommandException() {
         logic = new LogicManager(model, storage);
 
         // Execute add command
-        String addCommand = AddCommand.COMMAND_WORD + TYPE_DESC_AMY + NRIC_DESC_AMY + NAME_DESC_AMY + PHONE_DESC_AMY;
-        Person expectedPerson = new PersonBuilder(AMY).build();
+        String addCommand = AddCommand.COMMAND_WORD + TYPE_DESC_DOCTOR_AMY + NRIC_DESC_DOCTOR_AMY + NAME_DESC_DOCTOR_AMY
+                + PHONE_DESC_DOCTOR_AMY;
+        Person expectedPerson = new PersonBuilder(DOCTOR_AMY).build();
```

Comment:
Since we have the Doctor class and the DoctorBuilder, you could use them to create a Doctor object

Reply:
I suggest that that be done when we refactor Person into abstract class

Reply:
Noted